### PR TITLE
Update the README with non-Docker instructions

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,60 @@ You should be able to point your browser at
...
+Install `virtualenv`:
+
+```
+pip install virtualenv
+```
+
+Create and activate a new environment:
+
+```
+virtualenv venv
```
Comment (marcofucci) [inline README.md]: this should be

`virtualenv --python=python3 venv`

I guess?

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Pull request merged**